exercises/perfect-numbers/description.md

```diff
@@ -1,11 +1,10 @@
 # Description
 
-Determine if a number is perfect, abundant, or deficient based on
-Nicomachus' (60 - 120 CE) classification scheme for positive integers.
+Determine if a number is perfect, abundant, or deficient based on Nicomachus' (60 - 120 CE) classification scheme for positive integers.
 
 The Greek mathematician [Nicomachus][nicomachus] devised a classification scheme for positive integers, identifying each as belonging uniquely to the categories of **perfect**, **abundant**, or **deficient** based on their [aliquot sum][aliquot-sum].
 The aliquot sum is defined as the sum of the factors of a number not including the number itself.
-For example, the aliquot sum of 15 is (1 + 3 + 5) = 9
+For example, the aliquot sum of `15` is `1 + 3 + 5 = 9`
```

Comment:
```suggestion
For example, the aliquot sum of `15` is `1 + 3 + 5 = 9`.
```

The reason is that with the code markdown, it is now apparent (to me at least) we can end the sentence with a period, properly, and with no confusion (or at least less confusion).